### PR TITLE
Fix windows build

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: util/wren.mk [hunk 1]
```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```
Comment (munificent) [inline util/wren.mk]: This doesn't seem to be doing the right thing on Travis. Would `ifeq ($(OS),mingw32)` work here like some of the other Windows-specific tests use?

Comment (mattn) [inline util/wren.mk]: In default, %OS% is set as Windows_NT. AFAIK, I don't know the toolchain or make that set %OS% as mingw32.

Comment (munificent) [inline util/wren.mk]: >  I don't know the toolchain or make that set %OS% as mingw32.

`OS` isn't coming from the environment. It's defined by the makefile on line 81. :)

Comment (mattn) [inline util/wren.mk]: >OS isn't coming from the environment. It's defined by the makefile on line 81. :)

Did you make sure? See my screenshot. If you re-define in Makefile, it's not right way, I think.

Comment (munificent) [inline util/wren.mk]: I'm sure the makefile defines `OS`. I didn't realize at the time that collides with an existing Windows environment variable. The goal was never to use a value coming from the environment. The makefile wants to control the value of `OS`. We could give it another name if it's helpful.

Comment (mattn) [inline util/wren.mk]: Okay, I don't have strong objection. :)
What you want? Change the part to below?
```
ifeq ($(OS),mingw32)
```

Comment (munificent) [inline util/wren.mk]: Exactly right. :)

Comment (mattn) [inline util/wren.mk]: squashed. please take a look.

**Comment by munificent** [comment]
Awesome, thanks! :bird:

**Comment by mattn** [comment]
Thank you too.

**Pull request merged**